src/components/Home/MenuBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import theme from '../../styles/theme';
+
+interface MenuBarProps {
+  theme: typeof theme;
+}
+
+const MenuBar: React.FC<MenuBarProps> = () => {
+  return (
+    <St.MenuBarWrapper>
+      <St.LI>
+        <St.Menu className='home'>홈</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='event'>이벤트</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='movie-talk'>무비톡</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='fast-order'>패스트오더</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='gift-shop'>기프트샵</St.Menu>
+      </St.LI>
+      <St.LI>
+        <St.Menu className='cgv'>@CGV</St.Menu>
+      </St.LI>
+    </St.MenuBarWrapper>
+  );
+};
+
+export default MenuBar;
+
+const St = {
+  MenuBarWrapper: styled.div`
+        scrollbar-width: none;
+
+        overflow-x: auto;
+        flex-shrink: 0;
+
+        width: 37.5rem;
+        max-width: 100%;
+        height: 4.2rem;
+
+        white-space: nowrap;
+
+        background: ${({ theme }) => theme.colors['gradient']};
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+    `,
+
+  LI: styled.li`
+        display: inline-block;
+        margin-right: 1.8rem; 
+        list-style-type: none;
+    `,
+
+  Menu: styled.button`
+        color: ${({ theme }) => theme.colors['white']};
+        ${({ theme }) => theme.fonts['body_bold_15']};
```

Comment:
p1) 오 이렇게 대괄호로 접근해줘도 좋지만! 저는 string으로 바꿔주지 않아도 되어서 `. 접근자`로 사용하는것 같아요! 도윤이도 이렇게 스타일링 하는것 같아서.. 우리 아래처럼 theme 사용 방식 통일합시다 🖤
```suggestion
        color: ${({ theme }) => theme.colors.white};
        ${({ theme }) => theme.fonts.body_bold_15};
```

Reply:
넵넵 좋아요 통일해서 반영해뒀습니다✨